### DISCUSSION TITLE
Should we create multi docker images for multi-platform?

### DISCUSSION THREAD

**Discussion by vttranlina** [linagora/tmail-backend#753] [Ideas]
Idea from a man on mailing-list https://www.mail-archive.com/server-dev@james.apache.org/msg72976.html
With more "control permission" in tmail-backend project, I think we can create one more image (for arm64) and push it to linagora dockerHub.

**Comment by Arsnael** [comment]
That discussion was for apache james though... Is it stiiill valid for tmail at this stage?

**Comment by chibenwa** [comment]
IMO this deservedto be kept on Apache channels.

JIB should allow for this easily.

**Comment by vttranlina** [comment]
We did it in tmail-backend
https://github.com/linagora/tmail-backend/pull/874
